test/01LessonOneTest.js

```diff
@@ -4,34 +4,66 @@ var assert = require('assert');
 
 
 function returnTrue() {
+  return true;
     // should return true
-
 }
 
 function returnFalse() {
+  return false;
     // should return false
-
 }
 
 function greaterThan(big, small) {
+  if(big>small){
+    return true;
+  }
+    else {
+      return false;
+    }
+    var big = 10;
+    var small = 0;
```

Comment:
This doesn't seem to be causing problems, but I'd like you to get rid of these variable definitions here.